Services/CO.CDP.EntityVerification.Tests/Events/EventDeserializerTest.cs

```diff
@@ -10,6 +10,18 @@ public class EventDeserializerTest
     [Fact]
     public void ItDeserializesOrganisationRegisteredEvent()
     {
+        var organisationRegistered1 = new OrganisationRegistered()
+        {
+            Id = Guid.NewGuid(),
+            Name = "name",
+            Identifier = new Identifier() { Id = "1234", LegalName = "name", Scheme = "test" },
```

Comment:
Can we use examples more meaningful to the reader, please? I.e LegalName = "Acme" Scheme "GB-COH".

Reply:
Ah that was a mistake (can't think in this heat), it was there so I could get the json to do some end to end testing with local stack. Will remove it.